_posts/2016-05-26-Gerber.md

```diff
@@ -0,0 +1,306 @@
+---
+layout: post
+title: "Gerber Statistic Implementation in Rcpp and OpenMP"
+author: Yi King, Rafael Nicolas Fermin Cota, and Chris Chong - Next Level Analytics

```

Comment:
I believe Jekyll can index names but does (AFAIK) not know the language you invented here: `-- company`.  Please make it be the same as the other submission.  If you must, add a plug to your firm in the text.